src/chrome/content/rules/ArtStation.com.xml

```diff
@@ -0,0 +1,11 @@
+<ruleset name="ArtStation">
+	<target host="artstation.com" />
+	<target host="*.artstation.com" />
```

Comment:
Our style guide recommend to avoid wildcards. Can you document the reason why you used one here?